utilities/common.py

```diff
@@ -94,10 +94,11 @@ def common_parser():
     parser.add_argument("--trackerMuons", action='store_true', help="Use tracker muons instead of global muons (need appropriate scale factors too)")
     parser.add_argument("--binnedScaleFactors", action='store_true', help="Use binned scale factors (different helpers)")
     parser.add_argument("--onlyMainHistograms", action='store_true', help="Only produce some histograms, skipping (most) systematics to run faster when those are not needed")
-    parser.add_argument("-v", "--verbose", type=int, default=3, choices=[0,1,2,3,4],
-                        help="Set verbosity level with logging, the larger the more verbose (currently for setup_test_logger)");
     parser.add_argument("--met", type=str, choices=["DeepMETReso", "RawPFMET"], help="MET (DeepMETReso or RawPFMET)", default="RawPFMET")                    
     parser.add_argument("-o", "--outfolder", type=str, default="", help="Output folder")
+    parser.add_argument("--set-custom-logger", dest="setCustomLogger", action="store_true", default=False, help="Use logging with colors")
```

Comment:
If this works fine it can probably be true by default

Reply:
It does, but since it changes the color of the prompt based on the severity of the warnings it might print junks if you pipe into a file or something. In that case I could add a switch to set it false if you do so

Reply:
That's fine. You can always just add a --no-colors option or something.

Reply:
By add I mean replace this one

Reply:
Ok, I made the coloured logger the  default one, and I changed the option from  --set-custom-logger to --no-color-logger.
The advantage of the coloured logger is that it makes it easier to distinguish messages with different severity, so it can highlight warnings or errors (the info is printed with the nominal text color)